runtime/handler.go

```diff
@@ -176,6 +177,10 @@ func ForwardResponseMessage(ctx context.Context, mux *ServeMux, marshaler Marsha
 		return
 	}
 
+	if !doForwardTrailers {
+		w.Header().Set("Content-Length", strconv.Itoa(len(buf)))
+	}
```

Comment:
Mind adding a test for this to our integration tests? https://github.com/grpc-ecosystem/grpc-gateway/blob/main/examples/internal/integration/integration_test.go

Reply:
Hi @johanbrandhorst, I've pushed a change that updates the existing tests to check for the header